app/assets/javascripts/angular/controllers/topics_ctrl.coffee

```diff
@@ -15,6 +15,15 @@ angular.module "lean-coffee"
       if confirm('Are you sure you want to delete this?')
         topicsService.delete(topic)
 
+    @archive = (topic) =>
+      if confirm('Are you sure you want to archive this?')
+        topicsService.archive(topic)
+
+    @archiveAll = =>
+      if confirm('Are you sure you want to archive all discussed topics?')
+        topicsService.archiveAll()
+        topicsService.reload()
```

Comment:
Oh no! Our bad! I can't think of any reason why it would do that, though – really it shouldn't need to update anything at all when the page loads. :gasp: